src/plugins/embeddable/public/lib/embeddables/embeddable_action_storage.ts

```diff
@@ -17,32 +17,15 @@
  * under the License.
  */
 
+import { UiActionsActionStorage, UiActionsSerializedEvent } from '../../../../ui_actions/public';
 import { Embeddable } from '..';
 
-/**
- * Below two interfaces are here temporarily, they will move to `ui_actions`
- * plugin once #58216 is merged.
- */
-export interface SerializedEvent {
-  eventId: string;
-  triggerId: string;
-  action: unknown;
-}
-export interface ActionStorage {
-  create(event: SerializedEvent): Promise<void>;
-  update(event: SerializedEvent): Promise<void>;
-  remove(eventId: string): Promise<void>;
-  read(eventId: string): Promise<SerializedEvent>;
-  count(): Promise<number>;
-  list(): Promise<SerializedEvent[]>;
-}
-
-export class EmbeddableActionStorage implements ActionStorage {
+export class EmbeddableActionStorage implements UiActionsActionStorage {
   constructor(private readonly embbeddable: Embeddable<any, any>) {}
```

Comment:
```suggestion
  constructor(private readonly embbeddable: Embeddable) {}
```

Reply:
If we do this change there is an error:

![image](https://user-images.githubusercontent.com/9773803/76630846-f8850180-6540-11ea-87d7-0a7507fa24d3.png)